__tests__/app.test.js

```diff
@@ -363,5 +363,32 @@ describe("DELETE /api/comments/:comment_id/", () => {
     });
 })
 
+//Users
+
+describe("GET /api/users", () => {
+    it("200: responds with all users", () => {
+        return request(app)
+            .get("/api/users")
+            .expect(200)
+            .then((res) => {
+                expect(Array.isArray(res.body.users)).toBe(true);
```

Comment:
lets remove this test please. We are using a for Each below so that implicitly shows its an array